conf/battle/skill.conf

```diff
@@ -370,6 +370,13 @@ can_damage_skill: 1
 // Legacy Athena: 1
 land_protector_behavior: 0
 
+// NPC EMOTION behavior (Note 1)
+// On official servers, certain mobs cast NPC EMOTION skill which displays an emoticon and change their mode from
+// Aggressive to Passive for a certain time. The Athena behavior does not change their mode to Passive.
+// Official: 0
+// Legacy Athena: 1
+npc_emotion_behavior: 1
```

Comment:
Please, default this to 0.

Reply:
My bad, @Daegaladh, Commit 577dbf9 fix this.